lambdas/services/lloyd_george_validator.py

```diff
@@ -1,7 +1,44 @@
+import re
+
+from lambdas.models.nhs_document_reference import NHSDocumentReference
+
+
 class LGFileTypeException(ValueError):
-    """One or more of the files do not match the required file type."""
+    """One or more of the files do not match the required file type"""
     pass
 
-def validate_lg_file_type(file_type):
+class LGFileNameException(ValueError):
+    """One or more of the files do not match naming convention"""
+    pass
+
+class LGInvalidFilesException(Exception):
+    """One or more of the files are not valid"""
+    pass
+
+def validate_lg_file_type(file_type: str):
     if file_type != 'application/pdf':
         raise LGFileTypeException
+
+def validate_file_name(name: str):
+    lg_regex = r'[0-9]+of[0-9]+_Lloyd_George_Record_\[[A-Za-z]+\s[A-Za-z]+]_\[[0-9]{10}]_\[\d\d-\d\d-\d\d\d\d].pdf'
```

Comment:
Can I suggest to change the regex part for patient name to something more flexible like `[a-zA-Z ]+` ?  Because I think we can't be sure that every full name is made up of exactly 2 words. For me my legal name (on passport) is 3 words

Reply:
Done

Reply:
Will this A-Za-z include special characters such as e, è, é, ê, ë for people with names that include these?